### PR TITLE
Add Debug impl for ComparableListener

### PR DESCRIPTION
**Pull request by PLeVasseur**
Addresses [#154]

### CODE REVIEW

File: src/utransport.rs [hunk 1]
```diff
@@ -656,4 +674,26 @@ mod tests {
         let check_on_receive_res = up_client_foo.check_on_receive(&uuri_1, &umessage);
         assert!(check_on_receive_res.is_err());
     }
+
+    #[test]
+    fn test_comparable_listener_pointer_address() {
+        let bar = Arc::new(ListenerBar);
+        let comp_listener = ComparableListener::new(bar);
+        println!("comp_listener: {:?}", comp_listener);
```
Comment (sophokles73) [inline src/utransport.rs:682]: Do these println! statements contribute anything to the test?

Comment (PLeVasseur) [reply src/utransport.rs:682]: They let me easily see the result if I run with --nocapture to "prove" the Debug implementation works :slightly_smiling_face: 

Would it be better to use logging crate (tracing?) and then env_logger to dev-dependencies?

Comment (PLeVasseur) [reply src/utransport.rs:682]: I suppose I could do some string manipulation ensure we get the intended Debug implementation too, I suppose.

Which of those sounds reasonable?

Comment (sophokles73) [reply src/utransport.rs:682]: Well, I have stated before that IMHO expecting the output of Display::display to contain any particular text is an antipattern.

That said, if you want to make sure that _something_ is produced, you could do something like
```rust
let bar = Arc::new(ListenerBar);
let comp_listener = ComparableListener::new(bar);
assert!(!format!("{}", comp_listener).is_empty());
```

Comment (PLeVasseur) [reply src/utransport.rs:682]: Okay. I'll go with this.

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**